imap/index.c

```diff
@@ -5026,9 +5026,10 @@ EXPORTED int index_search_evaluate(struct index_state *state,
 }
 
 struct extractor_ctx {
-    struct backend *be;
     struct protstream *clientin;
-    unsigned failed;
+    char *hostname;
+    char *path;
+    struct backend *be;
```

Comment:
I've moved these into the context object, of which there's only one, and they get initialised at creation time